lib/node/node.ml

```diff
@@ -4,6 +4,7 @@ open Types
 module Message = Message
 module Client = Client
 module Failure_detector = Failure_detector
+module Inbox = Inbox
```

Comment:
Why?

Reply:
It's used in the deku integration branch i tested against, of this is wrong we should revert and fix downstream